packages/keystone-engine.vm/keystone-engine.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>keystone-engine.vm</id>
+    <version>0.9.2</version>
+    <authors>aquynh, genwei</authors>
+    <description>Keystone is a lightweight multi-platform, multi-architecture assembler framework.</description>
```

[nitpick] It may be useful to include it is a Python library in the description as there is no shortcut for it. I think we can do it while keeping the description short.
```suggestion
    <description>Keystone is a Python library providing a multi-platform, multi-architecture assembler.</description>
```